externals/kyuubi-spark-sql-engine/pom.xml

```diff
@@ -167,8 +167,18 @@
                         <includes>
                             <include>org.apache.kyuubi:kyuubi-common</include>
                             <include>org.apache.kyuubi:kyuubi-ha</include>
+                            <include>org.apache.curator:*</include>
```

Comment:
can we avoid using wildcard here?

Reply:
I tried to include curator-framework and curator-recipes here.

Let me wait the ut result and determine whether also include curator-test here.

Reply:
ok，include curator-client